user-form.html

```diff
@@ -176,6 +176,8 @@ <h2>Account Details</h2>
             <small>Error Message</small>
           </div>
 
+          <a href="https://github.com/Real-Dev-Squad/website-static"> Click here to complete image upload process </a>
```

Comment:
This is functional, no bugs as such but the user experience (UX) can be significantly improved by a slight modification
As per current implementation, the user will be taken to the website-static repo in the same tab and hence his form will remain incomplete, his data on the form will be lost. Please search on how you can make it open in new tab instead of current tab.

Reply:
Okay let me do that.